src/Azure.Functions.Cli/Common/Constants.cs

```diff
@@ -46,7 +47,7 @@ internal static class Constants
         {
             { WorkerRuntime.dotnet, new [] { "mcr.microsoft.com/azure-functions/dotnet", "microsoft/azure-functions-dotnet-core2.0", "mcr.microsoft.com/azure-functions/base", "microsoft/azure-functions-base" } },
             { WorkerRuntime.node, new [] { "mcr.microsoft.com/azure-functions/node", "microsoft/azure-functions-node8" } },
-            { WorkerRuntime.python, new [] { "mcr.microsoft.com/azure-functions/python", "microsoft/azure-functions-python3.6"  } },
+            { WorkerRuntime.python, new [] { "mcr.microsoft.com/azure-functions/python", "microsoft/azure-functions-python3.6", "microsoft/azure-functions-python3.7" } },
```

Comment:
Is that an actual image?

Reply:
Ahh, I confused this with `mcr.microsoft.com/azure-functions/python:2.0-python3.7-appservice`.
Will remove.